### PR TITLE
added ro RO translations

### PR DESCRIPTION
**Pull request by rusudinu**
## Description

Added romanian translations.

## Related Issues
-

## Checklist

- [x] I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I titled the PR using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0).
- [x] I did not modify the `CHANGELOG.md` nor the plugin version in `pubspec.yaml` files.
- [x] All existing and new tests are passing.
- [x] I have run the commands in `./scripts/before_push.sh` and it all passed successfully

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate that with a `!` in the title as explained in [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0)).
- [x] No, this is *not* a breaking change.

### CODE REVIEW

**Review by singerdmx — APPROVED** [review]

File: lib/src/l10n/generated/quill_localizations_ro.dart [hunk 1]
```diff
@@ -0,0 +1,545 @@
...
+
+/// The translations for Romanian Moldavian Moldovan (`ro`).
+class FlutterQuillLocalizationsRo extends FlutterQuillLocalizations {
+  FlutterQuillLocalizationsRo([super.locale = 'ro']);
+
+  @override
+  String get pasteLink => 'Lipește un link';
+
+  @override
+  String get ok => 'Ok';
```
Comment (ellet0) [inline lib/src/l10n/generated/quill_localizations_ro.dart:11]: This is not translated?

Comment (singerdmx) [reply lib/src/l10n/generated/quill_localizations_ro.dart:11]: Ok is probably universal?

Comment (rusudinu) [reply lib/src/l10n/generated/quill_localizations_ro.dart:11]: yeah, we don't have a translation for 'ok' in romanian

**Comment by ellet0** [comment]
I'm curios to know if you used `flutter gen-l10n` or `./scripts/regenerate_translations.sh`

**Comment by rusudinu** [comment]
i used flutter gen-l10n

**Pull request merged**